sdk/typescript/src/builder/TransactionBlock.ts

```diff
@@ -85,6 +85,18 @@ function createTransactionResult(index: number): TransactionResult {
 	}) as TransactionResult;
 }
 
+function isReceivingType(normalizedType: SuiMoveNormalizedType): boolean {
+	const tag = extractStructTag(normalizedType);
+	if (tag) {
+		return (
+			tag.Struct.address === '0x2' &&
```

Comment:
Not an issue for this PR, but it's interesting/concerning to me that the normalized representation of `0x2` isn't the fixed width (with leading zeroes) representation.  When we redesign the RPC, we want to standardise the representation of types, so we'll need to audit SDKs for this kind of dependency cc @hayes-mysten / @Jordan-Mysten